internal/repositories/repositories.go

```diff
@@ -0,0 +1,30 @@
+package repositories
+
+type LinkRepo interface {
+	GetLink(string) (string, bool)
+	CreateLink(string, string)
+	RemoveLink(string)
+}
+
+type memoryLink struct {
+	links map[string]string
+}
+
+func NewMemoryLink() LinkRepo {
+	return &memoryLink{
+		links: make(map[string]string),
+	}
+}
+
+func (m *memoryLink) GetLink(shortLinkID string) (originLink string, ok bool) {
+	originLink, ok = m.links[shortLinkID]
```

Comment:
не ошибка, можно без создания новых переменных
```suggestion
	return m.links[shortLinkID]
```